site/gatsby-site/src/contexts/LogHistoryContext.js

```diff
@@ -0,0 +1,64 @@
+import React, { createContext, useContext } from 'react';
+import { useMutation } from '@apollo/client/react/hooks';
+import { transformIncidentData } from '../utils/cite';
+import { transformReportData } from '../utils/reports';
+import { LOG_INCIDENT_HISTORY } from '../graphql/incidents';
+import { LOG_REPORT_HISTORY } from '../graphql/reports';
+import { stripMarkdown } from '../utils/typography';
+
+const LogHistoryContext = createContext({
+  logIncidentHistory(incident, user) {
+    incident;
+    user;
+  },
+  logReportHistory(report, updates, user) {
+    report;
+    updates;
+    user;
+  },
+});
+
+export default LogHistoryContext;
+
+export function LogHistoryContextProvider({ children }) {
```

Comment:
I think you don't need a context/provider, and just a `useLogHistory` function will do 🤔

Reply:
Done